solidstart/github-oauth/package.json

```diff
@@ -1,25 +1,26 @@
 {
-	"name": "username-and-password",
+	"name": "solidstart-github-oauth",
```

Comment:
Can you revert this change to make it consistent with the other examples?

Reply:
Reverted.